astroid/node_classes.py

```diff
@@ -274,77 +274,56 @@ class NodeNG:
     This is the base class for all Astroid node classes.
     """
 
-    is_statement = False
-    """Whether this node indicates a statement.
-
-    :type: bool
-    """
-    optional_assign = False  # True for For (and for Comprehension if py <3.0)
+    is_statement: ClassVar[bool] = False
+    """Whether this node indicates a statement."""
+    optional_assign: ClassVar[
+        bool
+    ] = False  # True for For (and for Comprehension if py <3.0)
     """Whether this node optionally assigns a variable.
 
     This is for loop assignments because loop won't necessarily perform an
     assignment if the loop has no iterations.
     This is also the case from comprehensions in Python 2.
-
-    :type: bool
     """
-    is_function = False  # True for FunctionDef nodes
-    """Whether this node indicates a function.
+    is_function: ClassVar[bool] = False  # True for FunctionDef nodes
```

Comment:
I don't understand why it's not `bool` here ?

Reply:
It is. Just additionally marked as ClassVar